src/CONST.js

```diff
@@ -474,7 +474,7 @@ const CONST = {
             SPLIT: 'split',
             REQUEST: 'request',
         },
-        AMOUNT_MAX_LENGTH: 10,
+        AMOUNT_MAX_LENGTH: 7,
```

Comment:
```suggestion
        AMOUNT_MAX_DECIMAL_LENGTH: 7,
```

Reply:
We can probably do something like  ` AMOUNT_INTEGER_PART_LIMIT`

Reply:
Can you revert the value to 10 please